index.html

```diff
@@ -24,7 +30,7 @@
     <div id='ammoElement'> </div>
     <div id='start-screen' style="padding: 320px; color:rgb(231, 228, 34)">
         <button type="button" onclick="handleStartGameBtnClick()">Let's Play!</button>
-       <div> <br><br><br> Сontrol keys:<br>
+       <div id="training"> <br><br><br> Сontrol keys:<br>
```

Comment:
Let's rename `training` -> `keyboard-controls-help`.

It took me a while to understand what is "training", and why do we hide it for touch devices :)

I believe hiding "keyboard-controls-help" on touch devices would sound better :)

Because touch devices have no keyboard 

![image](https://user-images.githubusercontent.com/6686679/215528174-1b12d8fd-903d-478f-9fef-43c197d8b9eb.png)